pkg/infra/iac3/templates/aws/rds_instance/factory.ts

```diff
@@ -49,7 +49,9 @@ function properties(object: aws.rds.Instance, args: Args) {
         }),
         RdsConnectionArn: pulumi.interpolate`arn:aws:rds-db:${region.name}:${accountId.accountId}:dbuser:${object.resourceId}/${object.username}`,
         Endpoint: object.endpoint,
-        ConnectionString: pulumi.interpolate`${args.Engine}://${object.username}:${object.password}@${object.endpoint.address}:${object.endpoint.port}/${args.DatabaseName}`,
+        ConnectionString: pulumi.interpolate`${args.Engine}://${object.username}:${object.password}@${object.endpoint}/${args.DatabaseName}`,
+        Host: object.address.apply((address) => address.split(':')[0]),
+        Port: object.address.apply((address) => address.split(':')[1]),
```

Comment:
Pretty sure `address` is only the dns name, I think you want `endpoint` here which has the port.